draft-wussler-openpgp-pqc.md

```diff
@@ -356,7 +356,7 @@ becomes available.\]
 
 ML-KEM [FIPS-203] is based on the hardness of solving the learning-with-errors
 problem in module lattices (MLWE). The scheme is believed to provide security
-against cryptanalytic attacks by classical as well as quantum computers. This
+against cryptoanalytic attacks by classical as well as quantum computers. This
```

Comment:
I disagree here, "cryptanalytic" or "cryptanalytis" are the commonly used term with a hits on Google while for "cryptoanalytic" I don't see any results on Google.

Reply:
agree with @falko-strenzke here, i tend to think cryptanalytic is the preferred form.